lib/src/shared/main/java/com/couchbase/lite/FileLogger.java

```diff
@@ -47,7 +47,10 @@
     private volatile LogLevel logLevel;
 
     // The singleton instance is available from Database.log.getFile()
-    FileLogger() { reset(); }
+    FileLogger() {
+        logLevel = LogLevel.NONE;
```

Comment:
can see logLevel set to NONE inside reset method, is there any specific reason why setting from here too?

Reply:
Hah!  Yeah.  The static analysis tools can't prove that the logLevel is @NonNull, otherwise!